src/10_functions.py

```diff
@@ -2,11 +2,20 @@
 
 # YOUR CODE HERE
 
+def is_even(x):
+  if x % 2 == 0 :
+    return bool(x)
+
```

Comment:
This is good too. But in this case, if `x` is not evenly divisible by 2 you can just return `True`. Else return `False` and you're good to go!

```
def is_even(x):
    if x % 2 == 0:
        return True:
    else:
        return False
```